### ISSUE TITLE
DOCHAZKA_DBI_ERR code is being replaced by error text, probably in call to mrest_declare_status()

### ISSUE THREAD

**Issue by smithfarm** [smithfarm/dochazka-rest#6]
The explanation should be "DOCHAZKA_DBI_ERR: DBI reports ..." but instead we get the "DBI reports ..." part *twice*:

```
Dochazka(2015-07-26) ncutler ACTIVE> interval 2015-04-18 - 2014-04-20
*** REST ERROR ***

Error encountered on attempted operation "Get intervals for employee ncutler (EID 41) in range ( 2015-04-18 00:00, 2014-04-20 24:00 )"
REST operation: GET interval/eid/41/( 2015-04-18 00:00, 2014-04-20 24:00 )
HTTP status: 500 Internal Server Error
Explanation: DBI reports DBD::Pg::db selectrow_array failed: ERROR:  range lower bound must be less than or equal to range upper bound at /usr/lib/perl5/site_perl/5.18.2/App/Dochazka/REST/Model/Shared.pm line 632.
: DBI reports DBD::Pg::db selectrow_array failed: ERROR:  range lower bound must be less than or equal to range upper bound at /usr/lib/perl5/site_perl/5.18.2/App/Dochazka/REST/Model/Shared.pm line 632.
Permanent? YES
```

**Comment by smithfarm** [comment]
Copied from https://rt.cpan.org/Ticket/Display.html?id=103827 - don't forget to close it there, too.

**Comment by smithfarm** [comment]
```
Dochazka(2015-07-26) root ADMIN> get interval self [ 2015-01-02, 2015-01-01 )
HTTP status: 500 Internal Server Error
Non-suppressed headers: {}
Response: {
  'payload' => {
                 'resource_name' => 'interval/self/:tsrange',
                 'http_method' => 'GET',
                 'http_code' => '500',
                 'permanent' => bless( do{\(my $o = 1)}, 'JSON::PP::Boolean' ),
                 'uri_path' => 'interval/self/[ 2015-01-02, 2015-01-01 )',
                 'found_in' => {
                                 'package' => 'Web::MREST::Entity',
                                 'line' => 250,
                                 'file' => '/usr/lib/perl5/site_perl/5.20.1/Web/MREST/Entity.pm'
                               }
               },
  'text' => 'DBI reports DBD::Pg::db selectrow_array failed: ERROR:  range lower bound must be less than or equal to range upper bound at /usr/lib/perl5/site_perl/5.20.1/App/Dochazka/REST/Model/Shared.pm line 632.',
  'code' => 'DBI reports DBD::Pg::db selectrow_array failed: ERROR:  range lower bound must be less than or equal to range upper bound at /usr/lib/perl5/site_perl/5.20.1/App/Dochazka/REST/Model/Shared.pm line 632.
',
  'level' => 'ERR',
  'rest_test' => 1,
  'http_message' => 'Internal Server Error',
  'http_code' => '500'
}
```

**Comment by smithfarm** [comment]
This happens on all DBI errors.

**Comment by smithfarm** [comment]
Fixed in `Web::MREST` 0.275:
Dochazka(2015-07-26) root ADMIN> get interval eid 1 [ 2000-1-2, 1999-1-2 )
HTTP status: 500 Internal Server Error
Non-suppressed headers: {
  'X-Web-Machine-Trace' => 'b13,b12,b11,b10,b9,b8,b7,b6,b5,b4,b3,c3,c4,d4,e5,f6,g7,g8,h10,i12,l13,m16,n16,o16,o18,o18b'
}
Response:
{
   "text" : "DBI reports DBD::Pg::db selectrow_array failed: ERROR:  range lower bound must be less than or equal to range upper bound at /usr/lib/perl5/site_perl/5.20.1/App/Dochazka/REST/Model/Shared.pm line 632.\n",
   "payload" : {
      "found_in" : {
         "package" : "Web::MREST::Entity",
         "file" : "/usr/lib/perl5/site_perl/5.20.1/Web/MREST/Entity.pm",
         "line" : 254
      },
      "http_code" : "500",
      "uri_path" : "interval/eid/1/[ 2000-1-2, 1999-1-2 )",
      "http_method" : "GET",
      "resource_name" : "interval/eid/:eid/:tsrange",
      "permanent" : true
   },
   "code" : "DOCHAZKA_DBI_ERR",
   "level" : "ERR",
   "rest_test" : 1
}
```
```
